cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -562,7 +567,63 @@ func (c *CStorPoolController) syncCsp(cStorPool *apis.CStorPool) {
 		c.recorder.Event(cStorPool, corev1.EventTypeWarning, string(common.FailureCapacitySync), string(common.MessageResourceFailCapacitySync))
 	} else {
 		cStorPool.Status.Capacity = *capacity
+
+		qn, err := convertToBytes([]string{capacity.Total, capacity.Free, capacity.Used})
```

Comment:
Remove 'else' case or better to put the changes in new fn

Reply:
done